lib/screens/stock_take_details/cash/cash_count.dart

```diff
@@ -110,6 +113,7 @@ class CashCount extends StatelessWidget {
           ),
         ),
         TextField(
+          keyboardType: keyboard,
```

Comment:
You just have to set the keyboard type here and not pass it as an argument.
I will fix it. Don't touch it.